lib/inv_pcg_quda.cpp

```diff
@@ -443,4 +457,6 @@ namespace quda
     return;
   }
 
+  std::unordered_map<std::string, std::vector<float>> MadwfAcc::host_training_param_cache; // empty map
```

Comment:
Having something like this in `PCG` implies that maybe we should introduce the `Acc` class abstraction, then a `MadwfAcc` that inherits it, sooner as opposed to later.

Reply:
I should move this to `madwf_ml.cpp` - I should have moved this there when I create `madwf_ml.cpp`.